docs/products/kafka/howto/tiered-storage-overview-page.md

```diff
@@ -1,72 +1,75 @@
+
 Aiven for Apache Kafka® offers a comprehensive overview of tiered storage, allowing you to understand its usage and make informed decisions.
 
 This overview page provides insights into various aspects of tiered storage, including
 hourly billing, settings, and storage details.
 
 ## Prerequisite
 
--   [Tiered storage enabled for the Aiven for Apache Kafka service](/docs/products/kafka/howto/enable-kafka-tiered-storage).
+- [Tiered storage enabled for the Aiven for Apache Kafka service](/docs/products/kafka/howto/enable-kafka-tiered-storage).
 
 ## Access tiered storage overview page
 
-1.  Log in to the [Aiven console](https://console.aiven.io/), choose
-    your project and select your Aiven for Apache Kafka service.
-1.  From the left sidebar, select **Tiered Storage**.
-    -   If tiered storage is not yet enabled for your service, you will
-        see the option to enabled it.
-    -   If tiered storage is enabled but not configured for any topics,
-        you have the option to set it up for topics directly. For more
-        details, see
-        [Enable and configure tiered storage for topics](/docs/products/kafka/howto/configure-topic-tiered-storage).
-1.  Once configured, you can view an overview of tiered storage and its
-    associated details.
+1. Log in to the [Aiven console](https://console.aiven.io/), choose
+   your project and select your Aiven for Apache Kafka service.
+1. Click <ConsoleLabel name="Tiered storage" />.
+
+   -   If tiered storage is not yet enabled for your service, you will
+       see the option to enabled it.
+   -   If tiered storage is enabled but not configured for any topics,
+       you have the option to set it up for topics directly. For more
+       details, see
+       [Enable and configure tiered storage for topics](/docs/products/kafka/howto/configure-topic-tiered-storage).
+1. Once configured, you can view an overview of tiered storage and its
+   associated details.
 
 ## Key insights of tiered storage
 
 Get a quick snapshot of the essential metrics and details related to
 tiered storage:
 
--   **Current billing expenses in USD**: Displays your tiered storage
-    costs, calculated at hourly rates.
--   **Forecasted month cost in USD**: Estimate your upcoming monthly
-    costs based on current usage.
--   **Remote tier usage in bytes**: View the volume of data that has
-    been tiered.
--   **Storage overview**: View how topics use
-    [remote storage](/docs/products/kafka/howto/tiered-storage-overview-page#remote-storage-overview).
+- **Current billing expenses in USD**: Displays your tiered storage
+  costs, calculated at hourly rates.
+- **Forecasted month cost in USD**: Estimate your upcoming monthly
+  costs based on current usage.
+- **Remote tier usage in bytes**: View the volume of data that has
+  been tiered.
+- **Storage overview**: View how topics use
+  [remote storage](/docs/products/kafka/howto/tiered-storage-overview-page#remote-storage-overview).
```

Comment:
Either use the 3rd person singular (e.g. "Displays") or the imperative (e.g. "View") consistently for all the list items. This comment refers to all similar lists in this article.